### PR TITLE
♻️ refactor(SuggestionModal): convert class based component to hooks

### PR DESCRIPTION
**Pull request by akashshyamdev**
PR for issue #65

### CODE REVIEW

**Comment by mikhael28** [comment]
@akashshyamdev currently get download speeds of 12kb/sec, and I haven't yet pulled down your repository. I would take hours to download the code and node_modules, so I will be reviewing later in the day. It's likely night time where you are anyway - will have an update on your two PR's soon! The one thing I would add, is that regarding the Prettier linting - I don't have any Prettier config, I literally just use the standard options on the base extension. Please run the linter on this file and recommit, if you haven't already - if you look at the diff for that commit, the spacing seems off (though that MIGHT be unrelated to the linter).

**Comment by mikhael28** [comment]
Upon opening the Suggestion modal, things are immediately broken. And looking at the code, you need to make the same updates to the state handleChange function - as in the other PR you sent in. It almost seems like you didn't truly test the code, before sending the pull-request :) Please make sure to always run the code and test it before submitting it for PR. You should never, ever send broken code in a PR.
Screen Shot 2021-09-03 at 5 36 05 PM

**Comment by akashshyamdev** [comment]
@mikhael28in which routes are these files displayed i.e. how do I get to this page?

**Comment by mikhael28** [comment]
@akashshyamdev The easiest way to find out where these files are displayed is to figure out where it's being mounted in the JSX code. You can use 'Command+Shift+F' and search for SuggestionModal to see in what file is that React component being mounted. Then, you can check the `Routes.js` to see what route that particular component is assigned to. Sometimes, you have to do a little bit of digging in the repository. For the sake of your learning, I'm not giving you the answer directly - I'm trying to teach you how to find it for yourself :D

As a friendly piece of open-source advice, ask this question before you send in a pull-request :) if you cannot confirm that you have working code, wait to send a pull-request. I appreciate the desire to contribute code, and you have been doing a really great job being productive the last couple of days.

File: src/context/SuggestionModal.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { Component } from "react";
+import React, { useState } from "react";
 import DialogContent from "@material-ui/core/DialogContent";
```
Comment (akashshyamdev) [inline src/context/SuggestionModal.js:2]: Test

**Comment by akashshyamdev** [comment]
Sorry about the bugs, I have fixed that now. Also, I've fixed up the UI. Thanks for the advice mate 👍

File: src/context/SuggestionModal.js [hunk 2]
```diff
@@ -98,26 +98,26 @@ export default function SuggestionModal({ schema, user, handleClose }) {
 
 	return (
 		<div>
-			<h1>Suggestion for {schema}</h1>
 			<DialogContent>
-				<DialogContentText>
+				<h1>Suggestion for {schema}</h1>
+				<DialogContentText style={{ fontSize: "24px" }}>
```
Comment (mikhael28) [inline src/context/SuggestionModal.js:103]: Can you make this font smaller, perhaps 18px?

File: src/context/SuggestionModal.js [hunk 3]
```diff
@@ -98,26 +98,26 @@ export default function SuggestionModal({ schema, user, handleClose }) {
...
 					to logo/image associated with the suggestion.
 				</DialogContentText>
 				<FormGroup controlId="title" bsSize="large">
-					<ControlLabel>Title</ControlLabel>
+					<ControlLabel style={{ fontSize: "16px" }}>Title</ControlLabel>
 					<FormControl value={formData.title} onChange={handleChange} />
 				</FormGroup>
 				<FormGroup controlId="description" bsSize="large">
-					<ControlLabel>Description</ControlLabel>
+					<ControlLabel style={{ fontSize: "16px" }}>Description</ControlLabel>
```
Comment (mikhael28) [inline src/context/SuggestionModal.js:112]: Make this 14px please

File: src/context/SuggestionModal.js [hunk 4]
```diff
@@ -98,26 +98,26 @@ export default function SuggestionModal({ schema, user, handleClose }) {
 
 	return (
 		<div>
-			<h1>Suggestion for {schema}</h1>
 			<DialogContent>
-				<DialogContentText>
+				<h1>Suggestion for {schema}</h1>
```
Comment (mikhael28) [inline src/context/SuggestionModal.js:102]: Can you add in the string formatting that you mentioned here? https://github.com/mikhael28/paretOS/issues/65
To make it capitalized, etc?

**Review by mikhael28 — CHANGES_REQUESTED** [review]
Great work - I'm now request a few UI changes, functionality things are good!

**Comment by akashshyamdev** [comment]
Ok @mikhael28, I've completed the changes :)

**Comment by mikhael28** [comment]
Great work, thank you for working through all the changes and reviews! Merging in :D

**Pull request merged**